HomeworkDay4/GitHubData.swift

```
@@ -0,0 +1,56 @@
+//
+//  GitHubData.swift
+//  HomeworkDay4
+//
+//  Created by Fumiaki Kobayashi on 2020/05/22.
+//  Copyright © 2020 Fumiaki Kobayashi. All rights reserved.
+//
+
+import Foundation
+
+class GitHubData {
+    //リポジトリ名
+    var repositoryName: String?
```

ここら辺のパラメータは let で optional じゃない方がいいです。
というのもこの `GitHubData` 自体は APIのレスポンスをモデル化したもので
絶対にあることが保証されてるプロパティだからです。

じゃあない場合はどうするのか、ということなんですがその場合はクラッシュでいいと思っています。
`Let it Crash` という思想ですね。

---

いつもvalueをoptionalにすべきか悩んでしまうところですが、```Let it Crash```思想参考になりました。
修正したいと思います。